packages/wrangler/templates/d1-beta-facade.js

```diff
@@ -1,4 +1,5 @@
 // src/shim.ts
+export * from "__ENTRY_POINT__";
```

Comment:
Needed to export any Durable Object classes that are defined as part of the script. It doesn't interfere with your default fetch export in this file.